src/test/resources/application.db.properties

```diff
@@ -1 +1,4 @@
-sql_select = Select * FROM %s where id = ?
\ No newline at end of file
+sql_select = Select * FROM %s where id = ?
```

Comment:
Why do we need these parameters as properties? Are we expecting that the end-user is going to modify them?